calico-enterprise/operations/ebpf/enabling-ebpf.mdx

```diff
@@ -221,5 +239,6 @@ To revert to standard Linux networking:
    ```bash
    kubectl patch ds -n kube-system kube-proxy --type merge -p '{"spec":{"template":{"spec":{"nodeSelector":{"non-calico": null}}}}}'
    ```
+2. If you are running MKE, download the cluster config as mentioned above, set `kube_proxy_mode` to `iptables` and `kube_default_drop_masq_bits ` to `false`
```

Comment:
Do we really need people to download, edit, and upload this configuration twice? Is there not a way to combine it in one step?

Reply:
This is under reversing the process, where we move from bpf to iptables.